subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/monitoring/CompositeTestIssuesMonitor.kt

```diff
@@ -66,7 +66,7 @@ class CompositeTestIssuesMonitor(
         return EventBuilder()
             .withTag(
                 "test_name",
```

Comment:
What does `test_name` mean?

Reply:
It's a key for value?

Reply:
`test_name` is a "className.$methodName" - unique test identifier
it's a key, yes